ADAL/tests/ADTest.pch

```diff
@@ -30,6 +30,9 @@
 #import <Foundation/Foundation.h>
 #import <XCTest/XCTest.h>
 #import "ADTestCase.h"
+#import "ADTestURLSession.h"
+#import "ADTestURLResponse.h"
+#import "ADTestAuthorityValidationResponse.h"
```

Comment:
It doesn't hurt for it to be, and I was also expecting to see it used in more of the integration test files. (It probably should be...)